bin/postinst-qa-check.d/50gnome2-utils

```diff
@@ -1,12 +1,12 @@
 # check for missing calls to gnome2-utils regen functions
 
 gnome2_icon_cache_check() {
-	local d f files=() find_args
+	local d f all_files=() find_args
```

Comment:
There's no point in including `find_args` here if you `local` it later.

Reply:
FIxed in 5fc7281657e2a5a4b22e969853364b142a37d05f, thanks.